### PR TITLE
add: homepage

### PR DESCRIPTION
**Pull request by nayyyhaa**
Added Homepage for quiz application.

Preview: https://innotion-quiz.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,11 +4,138 @@
...
+        <a class="m-l-3" href="/"><h3 id="logo">in.notion <span class="text-shd">Quiz</span></h3></a>
+        <ul class="nav-items row-flex no-bullet">
+          <li class="btn nav-link-btn h3 m-v-2 m-r-3"><a href="#">Leaderboard</a></li>
+          <li class="btn nav-link-btn h3 m-v-2 m-r-3"><a href="#">Rules</a></li>
+        </ul>
+        
+        <ul class="row-flex no-bullet">
+            <li class="dark-mode icon-toggle icon-btn rd-bdr grid-ctr colored-text m-r-3"></li>
+            <li class="nav-icon-btn icon-btn rd-bdr grid-ctr wt-text m-r-3">
+                <a href="#" class="grid-ctr">
```
Comment (keshavkumar-in) [inline index.html]: Use `javascript:;` in place of `#` it will not make your click go to top.

**Review by keshavkumar-in — APPROVED** [review]
UI is just so good, just added a suggestion. Other than that you are good to merge.

File: index.html (hunk 1 above)
Comment (nayyyhaa) [reply index.html]: made the changes.

File: styles/utils/font.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  font-family: "Great Vibes", cursive;
+  color: white;
+  font-size: 3rem;
+}
+
+#logo > * {
+  color: var(--COLOR-COMPLEMENTARY-LIGHT);
+  font-family: inherit;
+  font-size: inherit;
+}
```
Comment (anik31) [inline styles/utils/font.css]: suggestion - use classes instead of id

**Review by anik31 — APPROVED** [review]
Nice work @nayyyhaa 💯 The UI is so clean & it is good that you have added dark mode. Class naming & code quality is also good.

File: styles/utils/font.css (hunk 2 above)
Comment (nayyyhaa) [reply styles/utils/font.css]: Made the said changes. Thanks!

**Comment by nayyyhaa** [comment]
Thanks @keshavkumar-in & @anik31 for reviewing. Made the said changes, now merging the PR.

**Pull request merged**